drivers/windows/overlay/peerdb_windows.go

```diff
@@ -66,9 +72,15 @@ func (d *driver) peerAdd(nid, eid string, peerIP net.IP, peerIPMask net.IPMask,
 			return err
 		}
 
+		if system.GetOSVersion().Build == 14393 {
+			peerMu.Lock()
+		}
 		n.removeEndpointWithAddress(addr)
```

Comment:
@dani-docker : Thank you for pointing out that this function also needs to be under lock!  I'll double check all other functions

Reply:
cool!

Reply:
As far as I can tell, this should be good.